Framework/Core/Windows/Devices/Pwm/PwmPin.cs

```diff
@@ -0,0 +1,171 @@
+using System;
+using Windows.Devices.Pwm.Provider;
+
+namespace Windows.Devices.Pwm
+{
+    public sealed class PwmPin : IDisposable
+    {
+        private readonly int m_pinNumber;
+        private PwmController m_controller;
+        private IPwmControllerProvider m_provider;
+        private bool m_started = false;
+        private bool m_disposed = false;
+        private double m_dutyCycle = 0;

```

Comment:
Confirmed with UWP folks: Default behavior is no signal (always low) until overridden by a caller.